ament_tools/verbs/build/cli.py

```diff
@@ -100,6 +100,15 @@ def prepare_arguments(parser, args):
         '--start-with',
         help='Start with a particular package',
     )
+    parser.add_argument(
+        '--end-with',
+        help='End with a particular package',
+    )
+    parser.add_argument(
+        '--only-package',
+        '--only',
```

Comment:
It only works with single `-` options, switching the order would change the destination name. I'll leave as is (we discussed it off-line).